p4-16/spec/P4-16-spec.adoc

```diff
@@ -5646,9 +5647,10 @@ separately in the following two subsections.
 
 ==== Switch statement with `action_run` expression
 
-For this variant of `switch` statement, the expression must be of the
-form `t.apply().action_run`, where `t` is the name of a table (see
-<<sec-invoke-mau>>).  All switch labels must be names of
+This type of `switch` statement can only be used within control `apply`
+blocks.  For this variant of `switch` statement, the expression must be
+of the form `t.apply().action_run`, where `t` is the name of a table
+(see <<sec-invoke-mau>>).  All switch labels must be names of
```

Comment:
Isn't this implicit from the fact that `table.apply()`  can only be used in control apply blocks, not in functions or actions?

Reply:
"This type of `switch` statement can only be used within control `apply` blocks." is definitely implied by other parts of the spec, but it seems reasonable to say here, too.

Reply:
I also don't think it hurts to explicitly state that here, but if anyone strongly opposes it I can remove it.

The other clarifications are the most important parts of this PR.